src/components/grid-aware/TitleBlock/TitleBlock.tsx

```diff
@@ -1,9 +1,12 @@
-import PropTypes from "prop-types";
-import React from "react";
+import * as React from "react";
 
 import s from "./TitleBlock.module.css";
 
-const TitleBlock = ({ title }) => {
+type TitleBlockProps = {
```

Comment:
@richardxia Do we care about grammar? There is only one property in here yet the name is plural

Reply:
I feel like `props` is generally used to mean "the name of the object passed to a React component", regardless of how many properties it actually contains. Singular `prop` would, to me, suggest a specific key/value pair within the `props` object. I'd be thrown off if I saw it used to refer to the object itself, so I vote for keeping this plural by convention.

(It also makes it so we don't have to make trivial grammar updates when we add/remove props).

Reply:
Yeah, I agree with @jessypeck on both points. It's kind of like how the `children` prop on components is always `children` and not `child` even when there's only one child element passed in.

Reply:
LGTM then!